node/mock-test.ts

```diff
@@ -1,19 +1,27 @@
 import cp = require('child_process');
 import fs = require('fs');
-import path = require('path');
+import ncp = require('child_process');
 import os = require('os');
+import path = require('path');
 import cmdm = require('./taskcommand');
+import semver = require('semver');
 import shelljs = require('shelljs');
+import syncRequest = require('sync-request');
 
 const COMMAND_TAG = '[command]';
 const COMMAND_LENGTH = COMMAND_TAG.length;
+const downloadDirectory = path.join(process.env.HOME || process.env.HOMEPATH || process.env.USERPROFILE, 'azure-pipelines-task-lib', '_download');
```

Comment:
where does this go on windows? should we use appdatalocal?

Reply:
We could, I think this is probably fine. [HomePath](https://environmentvariables.org/HomePath) should resolve this, if not [UserProfile](http://environmentvariables.org/UserProfile) should, either way should point to Users\{username}